django/db/backends/postgresql/schema.py

```diff
@@ -12,6 +12,7 @@ class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
     sql_set_sequence_max = "SELECT setval('%(sequence)s', MAX(%(column)s)) FROM %(table)s"
 
     sql_create_index = "CREATE INDEX %(name)s ON %(table)s%(using)s (%(columns)s)%(extra)s"
+    sql_create_index_parameterized = "CREATE INDEX %(name)s ON %(table)s%(using)s (%(columns)s)%(with)s%(extra)s"
```

Comment:
Can we put `'%(with)s'` into `sql_create_index` and make it default to an empty string? I'd prefer not having an extra SQL template for this if possible.